Main.lean

```diff
@@ -56,14 +72,23 @@ def genCoreCmd := `[Cli|
   "Generate documentation for the core Lean modules: Init and Lean since they are not Lake projects"
 ]
 
+def bibPrepassCmd := `[Cli|
+  bibPrepass VIA runBibPrepassCmd;
+  "Run the bibliography prepass: copy the bibliography file to output directory"
+
+  ARGS:
+    source : String; "The bibliography file. If it is '-' or inexistent file, the reference page will be disabled."
```

Comment:
Why is this behavior chosen like so? What is the reason for having special support for a "-" file? Furthermore just silently disabling the reference page instead of throwing a proper error if passed a non existing file seems wrong to me. If the tool is told to act on a file that does not exist it should through an error.

To make this file optional, why not let the `lakefile` handle this situation and depending on whether the `references.bib` exists run a command or not.

Reply:
Then I need a command line switch for no bibliography files. Perhaps add a `--nobib` option? Or make the `source` optional, if it is absent, then disable bibliography?

Reply:
Please be explicit rather than implicit about it.

Reply:
Done. I added `--none`, `--json` and `--pybtex` flags to it.